cmd/translate/main_grpc_integration_test.go

```diff
@@ -419,6 +419,20 @@ func Test_ListServices_gRPC(t *testing.T) {
 
 // ------------------Messages------------------
 
+func randMessages() *translatev1.Messages {
```

Comment:
I believe this already exists in the main branch, please merge main into working branch.